### ISSUE TITLE
Key error when crawling cryptopunks data

### ISSUE THREAD

**Issue by williamchong** [a16z/nft-analyst-starter-pack#2]
I am trying to use this on `0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB` to crawl cryptopunks data
However a key error was thrown when running  `Updating ETH prices...` 

```
Process started for contract address: 0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB
Exporting token transfers...
Exporting logs...
Updating block-to-date mapping...
Updating ETH prices...
Traceback (most recent call last):
  File "/Users/william/nft-analyst-starter-pack/export_data.py", line 164, in <module>
    export_data()
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/william/nft-analyst-starter-pack/export_data.py", line 131, in export_data
    generate_sales_output(
  File "/Users/william/nft-analyst-starter-pack/core/generate_sales_output.py", line 41, in generate_sales_output
    transfers_df["date"] = transfers_df["block_number"].apply(
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/pandas/core/series.py", line 4433, in apply
    return SeriesApply(self, func, convert_dtype, args, kwargs).apply()
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/pandas/core/apply.py", line 1082, in apply
    return self.apply_standard()
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/pandas/core/apply.py", line 1137, in apply_standard
    mapped = lib.map_infer(
  File "pandas/_libs/lib.pyx", line 2870, in pandas._libs.lib.map_infer
  File "/Users/william/nft-analyst-starter-pack/core/generate_sales_output.py", line 42, in <lambda>
    lambda x: date_blocks_df.iloc[date_blocks_df.index.get_loc(x)]["date"]
  File "/Users/william/opt/anaconda3/lib/python3.9/site-packages/pandas/core/indexes/interval.py", line 637, in get_loc
    raise KeyError(key)
KeyError: 3919706```

**Comment by udaiparvatha** [comment]
I was trying to use this code for Adidas NFT Collection 

Process started for contract address: 0x28472a58A490c5e09A238847F66A68a47cC76f0f
Exporting token transfers...
Traceback (most recent call last):
  File "export_data.py", line 164, in <module>
    export_data()
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "export_data.py", line 88, in export_data
    export_token_transfers(
  File "/Users/viper/Documents/OneDrive/TellerFinance/nft-analyst-starter-pack/jobs/export_token_transfers.py", line 27, in export_token_transfers
    token_transfers = pd.read_csv(output)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 680, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1231, in _make_engine
    return mapping[engine](f, **self.options)
  File "/Users/viper/opt/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 75, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 551, in pandas._libs.parsers.TextReader.__cinit__
pandas.errors.EmptyDataError: No columns to parse from file

**Comment by emlazzarin** [comment]
There are two issues here. The first is that `date_block_mapping.csv` didn't have enough history, which is resolved by #5. The second is that this library only supports sales data from OpenSea right now. We have plans to add more sources.
